src/Network/Network/PrivateLinkService/PrivateLinkResource/GetAzurePrivateLinkResourceCommand.cs

```diff
@@ -89,6 +91,12 @@ public override void Execute()
                 this.Subscription = DefaultProfile.DefaultContext.Subscription.Id;
                 this.PrivateLinkResourceType = DynamicParameters[privateEndpointTypeName].Value as string;
             }
+
+            if (!GenericProvider.SupportsPrivateLinkResourceType(this.PrivateLinkResourceType))
+            {
+                throw new ArgumentException(string.Format(Properties.Resources.UnsupportPrivateLinkResourceType, this.PrivateLinkResourceType));
```

Comment:
AzPSArgumentException

Reply:
Use AzPSArgumentException need pass parameter name. We have to hardcode parameter name if use it.

Reply:
Please find a proper exception from https://github.com/Azure/azure-powershell-common/tree/main/src/Common/Exceptions

Reply:
Used AzPSApplicationException